doc/source/structures/misc/kosdelegate.rst

```diff
@@ -0,0 +1,77 @@
+.. _kosdelegate:
+
+KOSDelegate
+===========
+
+The structure `KOSDelegate` is what you get when you use the
+:ref:`Delegate at-sign syntax <kosdelegate_atsign>`, as in
+this example::
+
+    function myfunc { print "hello, there". }
+
+    local print_a_thing is myfunc@. // <--- Note the at-sign '@'.
+    // print_a_thing is now a KOSDelegate of myfunc.
+
+A KOSDelegate is a reference to the function that can be used to
+call the function later elsewhere in the code.
+
+The full explanation of the delegate feature
+:ref:`is explained elsewhere <delegates>`.  This page just 
+documents the members of the KOSDelegate structure for completeness.
+The full explanation of how this structure works and what it's
+for can be found on the :ref:`delegate page <delegates>`.
+
+Structure
+---------
+
+.. structure:: KOSDelegate
+
+    .. list-table:: Members
+        :header-rows: 1
+        :widths: 2 1 4
+
+        * - Suffix
+          - Type
+          - Description
+
+        * - :meth:`CALL(varying arguments)`
+          - same as the function this is a delegate for.
+          - calls the function this delegate wraps.
+        * - :meth:`BIND(varying arguments)`
+          - another KOSDelegate
+          - creates a new KOSDelegate with some arguments predefined.
+
+.. note::
+
+    This type is not serializable.
```

Comment:
When a type is not serializable I've been simply omitting this note. I think it could be potentially confusing, someone could have a quick glance, see a 'serializable-something' note and think it means that a type IS serializable.